patches/src/main/java/org/jclouds/docker/compute/options/DockerTemplateOptions.java

```diff
@@ -24,6 +24,7 @@
 
 import org.jclouds.compute.ComputeService;
 import org.jclouds.compute.options.TemplateOptions;
+import org.jclouds.docker.internal.NullSafeCopies;
```

Comment:
Definitely keep these files (the `jclouds-labs` PR in `patches`) as a separate commit